diff --git a/dist/getPublishPlan.mjs b/dist/getPublishPlan.mjs
index dc6ca95c3c26de55b3355d1f339b97ab79ba3d20..ce3de45a86b2d016328f45fcccc9c92601faeab2 100644
--- a/dist/getPublishPlan.mjs
+++ b/dist/getPublishPlan.mjs
@@ -511,7 +511,6 @@ const publish = async ({ pkg, release, tarballPath, interactive, otpCode }) => {
 		"--tag",
 		release.tag
 	];
-	if (!interactive) args.push("--json");
 	if (otpCode) args.push("--otp", otpCode);
 	const { exitCode, stdout, stderr } = await exec("yarn", args, {
 		nodePath: false,
